diff --git a/Project/QtCreator/QCTools.pro b/Project/QtCreator/QCTools.pro
index e7a840b..6afbe60 100755
--- a/Project/QtCreator/QCTools.pro
+++ b/Project/QtCreator/QCTools.pro
@@ -3,12 +3,9 @@ CONFIG += ordered
 
 SUBDIRS = \
         qctools-lib \
-        qctools-cli \
         qctools-gui
 
 qctools-lib.subdir = qctools-lib
-qctools-cli.subdir = qctools-cli
 qctools-gui.subdir = qctools-gui
 
-qctools-cli.depends = qctools-lib
 qctools-gui.depends = qctools-lib
